pkg/ocicni/ocicni.go

```diff
@@ -80,18 +80,17 @@ func buildFullPodName(podNetwork PodNetwork) string {
 // Lock network operations for a specific pod.  If that pod is not yet in
 // the pod map, it will be added.  The reference count for the pod will
 // be increased.
-func (plugin *cniNetworkPlugin) podLock(podNetwork PodNetwork) *sync.Mutex {
+func (plugin *cniNetworkPlugin) podLock(podNetwork PodNetwork) {
 	plugin.podsLock.Lock()
 	defer plugin.podsLock.Unlock()
-
 	fullPodName := buildFullPodName(podNetwork)
 	lock, ok := plugin.pods[fullPodName]
 	if !ok {
 		lock = &podLock{}
 		plugin.pods[fullPodName] = lock
 	}
+	lock.mu.Lock()
```

Comment:
The `lock` might already be created, so the reference count can only be incremented while `locked`.